src/renderer/components/TimelineSpace/Contents/FollowRequests.vue

```diff
@@ -16,7 +16,7 @@ import User from '@/components/molecules/User.vue'
 import { ACTION_TYPES } from '@/store/TimelineSpace/Contents/FollowRequests'
 
 export default defineComponent({
-  name: 'folllow-requests',
+  name: 'follow-requests',
```

Comment:
Notable.

(Reviewdog also hates me, but I had to make a change in order to let it speak... That's a separate problem.)

Reply:
OK